npm/webpack-dev-server-fresh/src/makeWebpackConfig.ts

```diff
@@ -49,8 +78,8 @@ export function makeWebpackConfig (
   config: CreateFinalWebpackConfig,
 ) {
   const { module: webpack } = config.sourceWebpackModulesResult.webpack
-  const userWebpackConfig = config.devServerConfig.webpackConfig as Partial<Configuration>
-  const frameworkWebpackConfig = config.frameworkConfig as Partial<Configuration>
+  const userWebpackConfig = modifyWebpackConfigForCypress(config.devServerConfig.webpackConfig as Partial<Configuration>)
```

Comment:
Do we have to do this twice or can we just modify the mergedConfig? I did something similar here: https://github.com/cypress-io/cypress/blob/ca87f90333aaae66711795dee95c034345aabc3b/npm/webpack-dev-server-fresh/src/makeWebpackConfig.ts#L55

Reply:
https://github.com/cypress-io/cypress/pull/20956/commits/a16b439dfba59176e991dbd777c90075e75319e5